Source/FlowEditor/Private/DetailCustomizations/FlowNode_CustomOutputDetails.cpp

```diff
@@ -1,93 +1,35 @@
 // Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
 
 #include "DetailCustomizations/FlowNode_CustomOutputDetails.h"
+#include "DetailLayoutBuilder.h"
 #include "FlowAsset.h"
-#include "Nodes/Route/FlowNode_CustomOutput.h"
-
-#include "DetailCategoryBuilder.h"
-#include "DetailWidgetRow.h"
-#include "PropertyEditing.h"
-#include "Widgets/Input/SComboBox.h"
-#include "Widgets/Text/STextBlock.h"
 
 #define LOCTEXT_NAMESPACE "FlowNode_CustomOutputDetails"
 
-void FFlowNode_CustomOutputDetails::CustomizeDetails(IDetailLayoutBuilder& DetailLayout)
+FFlowNode_CustomOutputDetails::FFlowNode_CustomOutputDetails()
 {
-	DetailLayout.GetObjectsBeingCustomized(ObjectsBeingEdited);
-	GetEventNames();
-
-	IDetailCategoryBuilder& Category = DetailLayout.EditCategory("CustomOutput", LOCTEXT("CustomEventsCategory", "Custom Output"));
-	Category.AddCustomRow(LOCTEXT("CustomRowName", "Event Name"))
-			.NameContent()
-				[
-					SNew(STextBlock)
-						.Text(LOCTEXT("EventName", "Event Name"))
-				]
-			.ValueContent()
-				.HAlign(HAlign_Fill)
-				[
-					SNew(SComboBox<TSharedPtr<FName>>)
-						.OptionsSource(&EventNames)
-						.OnGenerateWidget(this, &FFlowNode_CustomOutputDetails::GenerateEventWidget)
-						.OnSelectionChanged(this, &FFlowNode_CustomOutputDetails::PinSelectionChanged)
-						[
-							SNew(STextBlock)
-								.Text(this, &FFlowNode_CustomOutputDetails::GetSelectedEventText)
-						]
-				];
+	check(bExcludeReferencedEvents == false);
 }
 
-void FFlowNode_CustomOutputDetails::GetEventNames()
```

Comment:
_I renamed GetEventNames() to RebuildEventNames() because the "Get" naming implied it was a simple lookup (eg, accessor) but it actually did a lot of work and didn't return anything._